warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/EventMapper.java

```diff
@@ -699,7 +699,7 @@ public void processEvent(K1 key, RawRecordContainer value, List<DataTypeHandler<
             // Try to only parse the event once. Parse the event on the first pass and only if
             // the BaseIngestHelper class differs. The same class used by different handlers
             // *should* produce the same result.
-            if (null == previousHelper || !previousHelper.getClass().getName().equals(thisHelper.getClass().getName())) {
+            if (null == previousHelper || !thisHelper.getClass().isAssignableFrom(previousHelper.getClass())) {
```

Comment:
This is NOT what we want here.  We want to determine if this is the same class.  Instead we can change this with previousHelper.getClass().equals(thisHelper.getClass()).